DOC-1357 add desupported date for Serverless Standard (#312)
redpanda-data/cloud-docs: modules/get-started/pages/whats-new-cloud.adoc

== May 2025

=== Serverless Standard: deprecated

> Serverless Standard has been deprecated, and all existing clusters will be migrated to the new xref:get-started:cluster-types/serverless.adoc[Serverless] platform (with higher usage limits, 99.9% SLA, and additional regions) on August 31, 2025.

Reviewer: ```suggestion
Serverless Standard is deprecated. All existing clusters will be migrated to the new xref:get-started:cluster-types/serverless.adoc[Serverless] platform (with higher usage limits, 99.9% SLA, and additional regions) on August 31, 2025.
```
